djangocms_bootstrap4/contrib/bootstrap4_link/forms.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.forms import models, IntegerField, BooleanField, RadioSelect
```

Comment:
I suggest to import forms directly.
Like `from django import forms`
and change the usage of fields to be `forms.X_FIELD`